pkg/api/unversioned/types.go

```diff
@@ -162,6 +162,11 @@ const (
 	// Status code 409
 	StatusReasonConflict StatusReason = "Conflict"
 
+	// StatusReasonGone means the item is no longer available at the server and no
+	// forwarding address is known.
+	// Status code 410
+	StatusReasonGone StatusReason = "Gone"
```

Comment:
Good catch. Please include a change to that doc in this PR!

On Wed, Nov 11, 2015 at 2:27 PM, Chao Xu <notifications@github.com> wrote:

> In pkg/api/unversioned/types.go
> <https://github.com/kubernetes/kubernetes/pull/16628#discussion_r44596262>
> :
>
> > @@ -162,6 +162,11 @@ const (
> >  	// Status code 409
> >  	StatusReasonConflict StatusReason = "Conflict"
> >
> > +	// StatusReasonGone means the item is no longer available at the server and no
> > +	// forwarding address is known.
> > +	// Status code 410
> > +	StatusReasonGone StatusReason = "Gone"
>
> Hmm..Status code that will be sent by the server is listed In the
> api-converions doc, and 410 is not there.
> https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#http-status-codes.
> Does that make the status code part of the API?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/16628/files#r44596262>.
>